src/components/WorkQueueCombobox.vue

```diff
@@ -17,7 +17,7 @@
   import { useSubscription } from '@prefecthq/vue-compositions'
   import { computed, watch } from 'vue'
   import { useWorkspaceApi } from '@/compositions'
-  import { WorkQueue } from '@/models'
+  import { WorkPoolQueue } from '@/models'
```

Comment:
This PR completely removes the old `WorkQueue` model since to the server, they're both the exact same. The only meaningful differences between the types (not-server-side) have been removed in previous PR's barring the deprecated `workQueue.filter` -- which this PR also removes as it's 1) deprecated and 2) I couldn't find any usage of it anymore.


So throughout this PR, you'll see a lot of "renaming" of `WorkQueue` -> `WorkPoolQueue` for consistency.

The only places you should still see just `WorkQueue` is when deliberately working with queues w/o their parent work pool - i.e. get all work queues in workspace.